user/pages/04.Reference/10.get-quota-info/docs.en.md

```diff
@@ -116,11 +116,11 @@ compute.instances | Number of virtual machines (servers, instances) |
 compute.ram_mb | RAM for virtual machines in MiB |
 dns.zones | Number of DNS zones |
 network.floatingips | Number of floating IP addresses |
-network.lb_healthmonitors | Number of Neutron LBaaSv2 health monitors |
-network.lb_listeners | Number of Neutron LBaaSv2 listeners |
-network.lb_members | Number of Neutron LBaaSv2 pool members |
-network.lb_pools | Number of Neutron LBaaSv2 pools |
-network.loadbalancers | Number of Neutron LBaaSv2 load balancers |
+network.lb_healthmonitors | Number of Neutron LBaaS health monitors |
```

Comment:
Why did you remove the "v2"? There were to my knowledge two kinds of Neutron LBaaS and we never supported the Neutron LBaaS (v1).
In OpenStack Docs I consistently read "LBaaS v2" and we should probably insert the space but not omit the "v2" entirely.
https://wiki.openstack.org/wiki/Neutron/LBaaS/Deprecation mentions
Neutron-lbaas vs. LBaaS v2 API vs. Octavia vs. Octavia v2 API
For our purpose I'd suggest to distinguish between "Neutron LBaaS v2" and "Octavia LBaaS".

Reply:
added v2 again